### ISSUE TITLE
env:browser globals is missing KeyEvent

### ISSUE THREAD

**Issue by onemen** [eslint/eslint#5793]
**What version of ESLint are you using?**
2.6.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**
```js
module.exports = {
  "env": {
    "es6": true,
    "browser": true
  },

  "globals": {
  }
};
```

**What did you do? Please include the actual source code causing the issue.**
```js
"use strict";

function getEvent(aType, aOptions) {
  if (aType == "mouse")
    return new MouseEvent("click", getProp(aOptions));
  if (aType == "key")
    return new KeyboardEvent("click", getProp(aOptions));
  return undefined;
}

function getProp(aOptions) {
  let options = aOptions || {};
  return {
    button: options.button || 0,
    shiftKey: options.shift || false,
    ctrlKey: options.ctrl || false,
    altKey: options.alt || false,
    bubbles: true,
    cancelable: true,
    view: window,
  };
}

var code = {
  'get MouseEvent & KeyEvent': function(test) {
    test.ok((getEvent("mouse") instanceof MouseEvent), "event should be instanceof MouseEvent");
    test.ok((getEvent("key") instanceof KeyEvent), "event should be instanceof KeyEvent");
  },
};
```

**What actually happened? Please include the actual, raw output from ESLint.**
```sh
27:41  error  'KeyEvent' is not defined         no-undef
```

**Comment by platinumazure** [comment]
You're using KeyboardEvent in one place and KeyEvent in another. Is this intentional?

**Comment by onemen** [comment]
KeyboardEvent is instanceof KeyEvent

**Comment by gyandeeps** [comment]
@onemen Can you point us to docs for `KeyEvent` as I tried it in chrome and its undefined?

**Comment by onemen** [comment]
You are right, it look like KeyEvent is only available in Firefox
